docs/root/start/quick-start/run-envoy.rst

```diff
@@ -104,6 +104,46 @@ The ``-c`` or ``--config-path`` flag tells Envoy the path to its initial configu
                    -c /envoy-custom.yaml
          ...
 
+   .. tab:: Windows Service
+
+      You can start Envoy as Windows Service that is managed under `Windows Service Control Manager <https://docs.microsoft.com/en-us/windows/win32/services/using-services/>`_. 
+
+      First, you need to create the service. Assuming you have a custom configuration in the current directory named ``envoy-custom.yaml``. After you create the service you
+      can start it.
+      
+      From an **administrator** prompt run the following commands:
+
+      .. substitution-code-block:: console
+
+         > sc create EnvoyProxy binpath="envoy-static.exe --config-path envoy-demo.yaml"
+            [SC] CreateService SUCCESS
```

Comment:
Do we want to suggest an incomplete path to envoy-static.exe here? Adding a directory to the SCM's path list requires a reboot.

Reply:
> Do we want to suggest an incomplete path to envoy-static.exe here? 

No this wasn't my intention. I deliberately left it incomplete and the user would need to specify the full path to the executable or add `envoy-static` to the system path.

Reply:
@wrowe any suggestion here on how to improve the wording?

Reply:
Usually, I just use a syntax like _{path-to}_\Program.exe but I'm not sure what looks best in the rst docs. https://www.envoyproxy.io/docs/envoy/latest/operations/cli seems to indicate <path string> as envoy's preference.

Maybe C:\EnvoyProxy\envoy.exe is the best default and explain to replace C:\EnvoyProxy with the path to the envoy.exe binary?

I would drop all reference to PATH. User's won't grok the difference between their PATH and LocalSystem account PATH, or why PATH didn't update when they changed the systemwide path without rebooting, leading to avoidable trouble tickets. It's an unwise practice.

Reply:
Let's clearly mark this feature Experimental, until we round out the entire implementation with logging?

Reply:
> Usually, I just use a syntax like {path-to}\Program.exe but I'm not sure what looks best in the rst docs. https://www.envoyproxy.io/docs/envoy/latest/operations/cli seems to indicate as envoy's preference.

In the [Run Envoy](https://www.envoyproxy.io/docs/envoy/latest/start/quick-start/run-envoy) page they have it as just `envoy`. This implicitly assumes that envoy is in the path or in the current directory. 

I will go with "Maybe C:\EnvoyProxy\envoy.exe is the best default and explain to replace C:\EnvoyProxy with the path to the envoy.exe binary?"

> Let's clearly mark this feature Experimental, until we round out the entire implementation with logging?

Sure I will just add a note on that